### PR TITLE
Dev 4625 : add cut labels sdk examples

### PR DESCRIPTION
**Pull request by jude-hunt**
# Pull Request Checklist

- [x] Read the [contributing guidelines](https://github.com/finbourne/lusid-sdk-python-preview/blob/master/CONTRIBUTING.md)
- [x] Tests pass
- [ ] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: sdk/tests/tutorials/ibor/test_cut_labels.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+
+class CutLabels(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+
+        cls.scopes_api = lusid.ScopesApi(api_client)
```
Comment (MikeMcGarry) [inline sdk/tests/tutorials/ibor/test_cut_labels.py]: You don't use this scopes_api, so you can remove it.

File: sdk/tests/tutorials/ibor/test_cut_labels.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
```
Comment (MikeMcGarry) [inline sdk/tests/tutorials/ibor/test_cut_labels.py]: You create a lusid.PortfoliosApi twice. Here and on line 32. You don't use it anywhere so you can remove both lines.

File: sdk/tests/tutorials/ibor/test_cut_labels.py [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+        cls.transaction_portfolios_api = lusid.TransactionPortfoliosApi(api_client)
+        cls.property_definitions_api = lusid.PropertyDefinitionsApi(api_client)
```
Comment (MikeMcGarry) [inline sdk/tests/tutorials/ibor/test_cut_labels.py]: You don't use the property definitions API, remove it.

File: sdk/tests/tutorials/ibor/test_cut_labels.py [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+            # Add the codes of our cut labels to our dictionary
+            code[request.display_name] = request.code 
+            
+            # Send the request to LUSID to create the cut label
+            result = self.cut_labels.create_cut_label_definition(
+                create_request=request)
+
+            # Check that result gives same details as input
+            self.assertEqual(result.display_name, display_name) # should this be self?
```
Comment (MikeMcGarry) [inline sdk/tests/tutorials/ibor/test_cut_labels.py]: What do you mean here when you say "should this be self?"

File: sdk/tests/tutorials/ibor/test_cut_labels.py [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+        # define function to format cut labels
+        def cut_label_formatter(date, cut_label_code):
+            return str(date) + "N" + cut_label_code
+        
+        
+        ## Create cut labels
+        # create a dictionary to keep track of cut labels
+        code = {}
+
+        def create_cut_label(hours, minutes, display_name, description, time_zone):
```
Comment (MikeMcGarry) [inline sdk/tests/tutorials/ibor/test_cut_labels.py]: I'm not sure if it was the same in your notebook, if so I missed it, but you should avoid referring to global variables inside a function. Here you refer to "code" inside your function but you don't pass it in as a parameter. Either add each cut label to the dictionary outside the function or pass it into your function and return it.

File: sdk/tests/tutorials/ibor/test_cut_labels.py [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+        create_cut_label(hours=9, minutes=0, display_name="NYOpen", description="", time_zone="America/New_York")
+        create_cut_label(hours=17, minutes=0, display_name="NYClose", description="", time_zone="America/New_York")
+
+
+        ## Create portfolio
+        scope = 'cut_labels_demo'
+        portfolio_code = self.test_data_utilities.create_transaction_portfolio(scope)
+
+
+        ## Add instruments
```
Comment (MikeMcGarry) [inline sdk/tests/tutorials/ibor/test_cut_labels.py]: Perhaps "Get the instrument identifiers" or "Get the instruments" makes more sense here, this comment makes it sounds like you are adding them to LUSID.

Comment (jude-hunt) [reply sdk/tests/tutorials/ibor/test_cut_labels.py]: Done

File: sdk/tests/tutorials/ibor/test_cut_labels.py (hunk 4 above)
Comment (jude-hunt) [reply sdk/tests/tutorials/ibor/test_cut_labels.py]: Whoops, put this in when I was talking to myself and getting used to the syntax, I'll delete it

File: sdk/tests/tutorials/ibor/test_cut_labels.py (hunk 3 above)
Comment (jude-hunt) [reply sdk/tests/tutorials/ibor/test_cut_labels.py]: Done

File: sdk/tests/tutorials/ibor/test_cut_labels.py (hunk 2 above)
Comment (jude-hunt) [reply sdk/tests/tutorials/ibor/test_cut_labels.py]: Done

File: sdk/tests/tutorials/ibor/test_cut_labels.py (hunk 1 above)
Comment (jude-hunt) [reply sdk/tests/tutorials/ibor/test_cut_labels.py]: Done

File: sdk/tests/tutorials/ibor/test_cut_labels.py (hunk 5 above)
Comment (jude-hunt) [reply sdk/tests/tutorials/ibor/test_cut_labels.py]: Okay, I've added it as a parameter. Just checked my notebook and it's the same there, shall I change this as well?

Comment (MikeMcGarry) [reply sdk/tests/tutorials/ibor/test_cut_labels.py]: Sure, but not a priority, in the notebook less focus on the code

**Pull request merged**